js/ngDialog.js

```diff
@@ -7,7 +7,7 @@
 (function (root, factory) {
     if (typeof module !== 'undefined' && module.exports) {
         // CommonJS
-        module.exports = factory(require('angular'));
+        module.exports = require('angular');
```

Comment:
@voronianski Have you tested the code using browserify? I can confirm that something is not correct here.